deepwell/src/services/text.rs

```diff
@@ -150,8 +150,15 @@ impl TextService {
                     .add(not_in_column!(
                         MessageRecord,
                         message_record::Column::CompiledHash,
+                    ))
+                    .add(not_in_column!(
+                        Alias::new("forum_post_revision"),
+                        Alias::new("wikitext_hash"),
+                    ))
+                    .add(not_in_column!(
+                        Alias::new("forum_post_revision"),
+                        Alias::new("compiled_html_hash"),
```

Comment:
Ah I see, temporary measure until new `src/models` files are generated. I'll do that in a separate PR because the autogen tool needs some manual correction unfortunately.